Basic-Recruit-Training/Person.cpp

```diff
@@ -1,17 +1,18 @@
-#include "Person.h"
 #include <iostream>
 #include <string>
+#include "Person.h"
 
 Person::Person()
 {
-    age_= new int(0);
-    name_= "";
+    age_ = new int(0);
+    name_ = "";
 }
 
 Person::Person(const std::string& name, int age)
+: name_(name),
+age_(new int(age))
```

Comment:
Just to line it up, the comma on line 12 should be on line 13, so
`: name_(name)`
`, age_(new int(age))`